packages/coordinator/scripts/uploadRoundMetadata.ts

```diff
@@ -0,0 +1,161 @@
+import { put } from "@vercel/blob";
```

Comment:
wouldn't it be easier to pass this as a json file? so that it can be sent over from the frontend in a post request and uploaded

Reply:
I think both works? here I call `put` to upload an object to the vercel storage as a json file, and the returned file link is stored in the contract when initializing, which means people would fetch a json file when they wanna know the metadata of this round. The outcome is the same as the method you said.